spec/spec_helper.rb

```diff
@@ -8,11 +8,7 @@
 ActiveRecord::Migration.verbose = false
 
 require 'combustion'
-Combustion.initialize! :active_record do
-  if config.active_record.sqlite3.respond_to?(:represent_boolean_as_integer)
-    config.active_record.sqlite3.represent_boolean_as_integer = true
-  end
-end
```

Comment:
The `represent_boolean_as_integer` config option is deprecated in Rails 6.0.

Reply:
Do we still need this? it seems like allowing rails to track booleans however it wants is fine.

Reply:
Yeah, we don't need that option.  The deprecation notice said it will always be true, so it won't change anything for us anyway.